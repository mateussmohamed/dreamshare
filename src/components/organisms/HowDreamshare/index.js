import React from 'react'

import { Container, Row, Column, DreamshareTitle } from 'atoms'
import { DreamshareStep } from 'molecules'

const HowDreamshare = () => (
  <Container>
    <DreamshareTitle>How Dreamshare</DreamshareTitle>
    <Row flexWrap="wrap" mx={0}>
      <Column width={[1, 1 / 3, 1 / 3]} flex="none">
        <DreamshareStep
          badge="Step 1"
          title="Sed leo enim, condimentum"
          description="Quisque libero libero, dictum non turpis in, luctus semper lorem. Donec rhoncus a leo sit amet facilisis."
          image="/assets/images/planing-trip.jpg"
        />
      </Column>
      <Column width={[1, 1 / 3, 1 / 3]} flex="none">
        <DreamshareStep
          badge="Step 2"
          title="Morbi velit risus"
          description="Nulla venenatis tempor dui in molestie. Nulla quis dictum purus, sit amet porttitor est."
          image="/assets/images/road-trip.jpg"
        />
      </Column>
      <Column width={[1, 1 / 3, 1 / 3]} flex="none">
        <DreamshareStep
          badge="Step 3"
          title="Sed leo enim, condimentum"
          description="Quisque libero libero, dictum non turpis in, luctus semper lorem. Donec rhoncus a leo sit amet facilisis."
          image="/assets/images/join-trip.jpg"
        />
      </Column>
    </Row>
  </Container>
)

export default HowDreamshare
