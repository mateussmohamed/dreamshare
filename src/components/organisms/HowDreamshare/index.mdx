---
name: HowDreamshare
menu: Organisms
---

import { Playground, PropsTable } from 'docz'
import HowDreamshare from './index'

# HowDreamshare
<PropsTable of={HowDreamshare} />

<Playground>
  <HowDreamshare />
</Playground>
