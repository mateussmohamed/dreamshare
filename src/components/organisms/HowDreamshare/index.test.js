import React from 'react'
import { shallow } from 'enzyme'
import 'jest-styled-components'

import HowDreamshare from './index'

describe('[<HowDreamshare />]', () => {
  const wrap = (props = {}) => shallow(<HowDreamshare {...props} />).dive()

  test('matches to snapshot', () => {
    const wrapper = wrap()

    expect(wrapper).toMatchSnapshot()
  })
})
