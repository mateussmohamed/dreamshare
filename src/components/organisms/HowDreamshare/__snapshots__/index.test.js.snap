// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[<HowDreamshare />] matches to snapshot 1`] = `
<Box
  css={
    Object {
      "maxWidth": "1024px",
    }
  }
  mx="auto"
>
  <HowDreamshare__Title>
    How Dreamshare
  </HowDreamshare__Title>
  <Row
    flexWrap="wrap"
    mx={0}
  >
    <Column
      flex="none"
      width={
        Array [
          1,
          0.3333333333333333,
          0.3333333333333333,
        ]
      }
    >
      <DreamshareStep
        badge="Step 1"
        description="Quisque libero libero, dictum non turpis in, luctus semper lorem. Donec rhoncus a leo sit amet facilisis."
        image="/assets/images/planing-trip.jpg"
        title="Sed leo enim, condimentum"
      />
    </Column>
    <Column
      flex="none"
      width={
        Array [
          1,
          0.3333333333333333,
          0.3333333333333333,
        ]
      }
    >
      <DreamshareStep
        badge="Step 2"
        description="Nulla venenatis tempor dui in molestie. Nulla quis dictum purus, sit amet porttitor est."
        image="/assets/images/road-trip.jpg"
        title="Morbi velit risus"
      />
    </Column>
    <Column
      flex="none"
      width={
        Array [
          1,
          0.3333333333333333,
          0.3333333333333333,
        ]
      }
    >
      <DreamshareStep
        badge="Step 3"
        description="Quisque libero libero, dictum non turpis in, luctus semper lorem. Donec rhoncus a leo sit amet facilisis."
        image="/assets/images/join-trip.jpg"
        title="Sed leo enim, condimentum"
      />
    </Column>
  </Row>
</Box>
`;
